metricbeat/module/elasticsearch/shard/_meta/data.json

```diff
@@ -1,40 +1,43 @@
 {
     "@timestamp": "2017-10-12T08:05:34.853Z",
-    "beat": {
-        "hostname": "host.example.com",
-        "name": "host.example.com"
-    },
     "elasticsearch": {
         "cluster": {
-            "id": "91RpCx2xSQ21pVPTZfDK0Q",
-            "name": "elasticsearch",
+            "id": "tMjf3CQ_TyCXNfcoR9eTWw",
+            "name": "docker-cluster",
             "state": {
-                "id": "MBE4XrQOSf6ScXRTuCO1Pw"
+                "id": "n-UoXaqYRoOe9qAC76IG6A"
             }
         },
         "index": {
-            "name": "heartbeat-7.0.0-alpha1-2018.08.27"
+            "name": ".apm-custom-link"
         },
         "node": {
-            "name": "Z4hBonPxQVW9qPKEHpwWCg"
+            "name": "hx-oJ1-aT_-5pRG22JMI1Q"
         },
         "shard": {
             "number": 0,
             "primary": true,
             "relocating_node": {
                 "name": null
             },
+            "source_node": {
+                "name": "1fb2aa83efac",
+                "uuid": "hx-oJ1-aT_-5pRG22JMI1Q"
+            },
```

Comment:
I did not see the `source_node.*` fields mentioned in the metricset's `fields.yml`. Should they be?

Reply:
Good catch! I completely forgot that I have to deactivate the test to check if all fields were documented.